long_term/load.py

```diff
@@ -0,0 +1,43 @@
+"""Given a file name uploads local json file to S3 bucket."""
+from os import environ as ENV
+from datetime import date
+
+from dotenv import load_dotenv
+from boto3 import client
+import pandas as pd
+
+
+load_dotenv()
+
+
+def create_csv(df: pd.DataFrame) -> str:
+    """Creates local csv from pandas dataframe."""
+    filename = f"summary_{date.today()}.csv"
```

Comment:
The file name would be "summary_YYYY-MM-DD.csv", could we turn the "_" to a "-" for uniqueness?